src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.lock.json

```diff
@@ -275,6 +275,9 @@
       "Microsoft.NETCore.ConsoleHost/1.0.0-beta-23328": {
         "type": "package"
```

Comment:
go ahead and delete this lock file as we are trying to avoid having the checked in now.